tests/integration/resources/_schema.json

```diff
@@ -7,7 +7,7 @@
     },
     "numstr": {
       "type": "string",
-      "index": 1,
+      "index": 2,
```

Comment:
I'll bet there was a story behind finding this!

Reply:
Hah, I think it might have just been red herrings while I was originally working on a NULL integration test, but figured it might as well get fixed before I forget again...